docs/unlocking_the_bootloader.md

```diff
@@ -0,0 +1,32 @@
+#### On unlocking the bootloader
+Devices by *Samsung, Google, OnePlus, Fairphone* and others make it relatively easy to unlock the bootloader and get good support in the installer.
+
+For some devices, it is necessary to unlock the bootloader manually. You usually need to create an account on the manufacturer's website, wait for a certain time and obtain a code from there. OpenAndroidInstaller will try to guide you as far as possible. These vendors include *Sony, Motorola and Xiaomi* amongst others.
+
+Other phone manufacturers no longer allow you to unlock the bootloader. Nothing can be done if you have not unlocked your device in time. These manufacturers include *Huawei, Honor, LG and ASUS*. Support for these manufacturers will always be very limited.
+
+<details>
+    <summary>
+        <b>Here is a brief overview of the bootloader policies of some of the most popular brands (may be incomplete/inaccurate)</b>
+    </summary>
+
+| Brand | Flashing tool | Unlocking Method | Loss of guarantee\* | Supported models |
```

Comment:
What is this referring to? Support for unlocking or the installer?
Not sure if we want to maintain a list of devices that are bootloader unlockable or not.

Reply:
To what exactly are you referring to?

Reply:
Ohhh do you mean the `Supported models`?
This is just the data I took from the website linked below and combined it with my own and the internet's data... :D 
There is not really a need to actively maintain it, as not much will change in the future, as the listed brands either allow or don't allow unlocking (anymore)...